source/content/guides/decoupled/overview/04-site.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pantheon Front-End Sites
+subtitle: Site Creation
+description: Create and configure a Front-End Site.
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan, backlineint, cobypear, hckia]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/overview/site
+anchorid: site
+reviewed: "2023-03-16"
+contenttype: [guide]
+innav: [false]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides configuration information for Front-End Sites.
+
+## Prerequisites
+
+* The **Decoupled** tab is enabled in your dashboard after onboarding.
+
+* Your site repository is in GitHub.
+
+* You have a domain to use for the Front-End site.
+
+* You will use Next.js or Gatsby as the frontend framework.
```

Comment:
Is there a way we can signal that any JavaScript based runtime, or any static site generator that outputs html/css/js is useable, but we only fully support Next.js and Gatsby?

Reply:
Added!